lib/ASRouter.jsm

```diff
@@ -116,6 +126,12 @@ const LOCAL_MESSAGE_PROVIDERS = {
 };
 const STARTPAGE_VERSION = "6";
 
+// Remote Settings
+const RS_SERVER_PREF = "services.settings.server";
+const RS_MAIN_BUCKET = "main";
+const RS_COLLECTION_L10N = "ms-language-packs"; // "ms" stands for Messaging System
```

Comment:
was the `ms` prefix really necessary then? It's sad that it requires a comment ;)

Reply:
I figured this collection was a "Messaging System" specific at the moment. I don't mind making it go as "language-packs" though :), I believe it's fine to rename a collection at this point, correct?

Reply:
I'd prefer if we kept this to be messaging system specific for now.

When we add actual language packs in remote settings, we'll likely want things to change, also based on what we're learning here.